pcdsdevices/mv_interface.py

```diff
@@ -173,8 +187,13 @@ def umvr(self, delta, timeout=None):
     def mv_ginput(self, timeout=None, wait=True):
         """
         Moves to a location the user clicks on.
+
+        If there are existing plots, this will be the position on the most
+        recently active plot. If there are no existing plots, an empty plot
+        will be created with the motor's limits as the range.
         """
-        print("Select new motor x-position in current plot by mouseclick")
+        logger.info(("Select new motor x-position in current plot "
```

Comment:
Duplicated parentheses

Reply:
Somehow I thought that you needed the inner parentheses to concat the multiline string, but I am mistaken.